modules/oadp-deleting-backup-repository.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-deleting-backups-doc.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-deleting-backup-repository_{context}"]
+= Deleting a backup repository
+
+After you delete the backup, and after the Kopia repository maintenance cycles to delete the related artifacts, are complete, there is no metadata, or manifest objects referencing the backup. You can then delete the `backuprepository` custom resource (CR) to complete the backup deletion process.
+
+.Prerequisites
+
+* You have deleted the backup of your application.
+* You have waited up to 72 hours after the backup is deleted. This time frame allows Kopia to run the repository maintenance cycles.
+
+.Procedure
+
+. To get the backup repository CR for a backup, run the following command:
```

Comment:
Could be a little more specific here about what they'll be looking for in the output:
```suggestion
. To get the name of the backup repository CR for a backup, run the following command:
```